javascript/javascript1/week2/class07Students.js

```diff
@@ -0,0 +1,52 @@
+// copy your homework solution here to make the webpage work!
+
+const class07Students = [];
+let count = 0;
+function addStudentToClass(studentName) {
+   if (studentName.toLowerCase() == 'queen') {
```

Comment:
Nice use of toLowerCase

Reply:
Since you're not checking if the name already exists first, "Queen" can be added multiple times.